docs/deployment/getting_started_locally.md

```diff
@@ -12,6 +12,18 @@ Based on [Skaffold](https://skaffold.dev/), the container images for all require
 
 ![Architecture Diagram](content/getting_started_locally.png)
 
+## Alternatives
+
+When deploying Gardener on your local machine you might face several limitations:
+
+- your machine doesn't have enough compute resources (see [prerequisites](#prerequisites)) for hosting a second seed cluster or multiple shoot clusters
+- testing Gardener's [IPv6 features](../usage/ipv6.md) requires a Linux machine and native IPv6 connectivity to the internet, but you're on macOS or don't have IPv6 connectivity in your office environment or via your home ISP
+
+In these cases, you might want to check out one of the following guides that run the setup described in this guide elsewhere for circumventing these limitations:
```

Comment:
```suggestion
In these cases, you might want to check out one of the following options that run the setup described in this guide elsewhere for circumventing these limitations:
```
Nit to avoid repetition.